helssa/settings.py

```
@@ -0,0 +1,122 @@
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-wpdo026hg-=b0!7x-njz@*e)ux#g+ua0l&wf%5x#i81*la%0-h'
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)